x/mint/README.md

```diff
@@ -47,6 +30,74 @@ oneYear := time.Hour * 24 * 365
 | 19   | 1.50              |
 | 20   | 1.50              |
 
+- **Year** indicates the number of years elapsed since chain genesis.
+- **Inflation (%)** indicates the percentage of the total supply that will be minted in the next year.
+
+## Terms
+
+- **Inflation Rate**: The percentage of the total supply that will be minted each year. The inflation rate is calculated once per year on the anniversary of chain genesis based on the number of years elapsed since genesis. The inflation rate is calculated as `InitialInflationRate * ((1 - DisinflationRate) ^ YearsSinceGenesis)`. See [./types/constants.go](./types/constants.go) for the constants used in this module.
+- **Annual Provisions**: The total amount of tokens that will be minted each year. Annual provisions are calculated once per year on the anniversary of chain genesis based on the total supply and the inflation rate. Annual provisions are calculated as `TotalSupply * InflationRate`
```

Comment:
[Question] Does this `TotalSupply` refer to the initial total supply? or the total supply plus all the tokens minted throughout the past years?